docs/cli/index.mdx

```diff
@@ -138,39 +137,49 @@ Enable auto-completion using the following commands.
 
 ### zsh auto-completion
 
-<!-- TODO: add more information about zsh to make comparable to bash section -->
+1. Add the following line to your `~/.zshrc` file:
 
-Add the following code snippet to your `~/.zshrc` file:
+   ```sh
+   eval "$(temporal completion zsh)"
+   ```
 
-```sh
-source <(temporal completion zsh)
-```
+2. Re-launch your shell or run:
 
-If you're running auto-completion from the terminal, run the following command:
+   ```sh
+   source ~/.zshrc
+   ```
 
-```sh
-echo 'source <(temporal completion zsh)' >> ~/.zshrc
-```
+### Bash auto-completion
 
-After setting the variable, run the following command:
+1. Install [bash-completion](https://github.com/scop/bash-completion#installation) and add the software to your `~/.bashrc`.
 
-`source ~/.zshrc`.
+2. Add the following line to your `~/.bashrc` file:
 
-### Bash auto-completion
+   ```sh
+   eval "$(temporal completion bash)"
```

Comment:
Comment: tested this with the default Apple Bash install, and found it errored during completion. I leave my system as "vanilla" as possible, and as close to the default installs that people will encounter when first using their computer (in this case macOS Sonoma, Bash).

```
bash-3.2$ temporal serbash: _get_comp_words_by_ref: command not found
ver
```

Reply:
Do you have `bash-completion` installed? This error usually means the completion library isn't loaded in your shell. I can add a note on that in the docs.

Reply:
👍 I tested it too! And apparently had bash-completion installed even though I use zsh, heh. I blame Homebrew. :)